Ms2Database/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Ms2Database.DB_Classes;
+
+namespace Ms2Database
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            using (Ms2DBContext context = new Ms2DBContext())
+            {
+                Account account = new Account()
+                {
+                    Username = "localhost",
+                    Password = ""
+                };
+                context.Accounts.Add(account);
+                context.SaveChanges();
+            }
```

Comment:
Can you drop a `Console.WriteLine` saying it ran just so people know something happened?